packages/grafana-ui/src/components/uPlot/paths.ts

```diff
@@ -0,0 +1,257 @@
+import uPlot, { Series } from 'uplot';
+
+export function Bars(u: uPlot, seriesIdx: number, idx0: number, idx1: number): Series.Paths {
```

Comment:
In general classes/components get exported as uppercase and functions/objects are lowercase.

Since this has to match a known type, should we export it as:
```suggestion
export const barsBuilder: Series.PathBuilder = (u: uPlot, seriesIdx: number, idx0: number, idx1: number) => {```